.github/workflows/ci.yml

```diff
@@ -67,7 +69,9 @@ jobs:
     - name: Install
       run: pip install -e . -v
     - name: fetch metadata
-      run: aiida-registry fetch
+      env:
+        GITHUB_TOKEN: ${{ secrets.TOKEN }}
+      run: aiida-registry fetch --token $GITHUB_TOKEN
```

Comment:
Note: you don't even necessarily need to create a commandline option for it, you can simply read the environment variable via `os.environ` if it exists.